dojo/templates/dojo/custom_pdf_report_endpoint_list.html

```diff
@@ -112,17 +112,23 @@ <h6>Vulnerable Endpoints / Systems</h6>
                                         </div>
 
                                         {% if endpoints %}
-                                            {% colgroup endpoints into 2 cols as grouped_items %}
-                                            <table class="table-striped table table table-condensed table-hover finding-endpoints">
-                                                {% for row in grouped_items %}
-                                                    <tr>
-                                                        {% for item in row %}
-                                                            <td>{% if item %}<i class="fa fa-circle"></i>
-                                                                {{ item }}{% endif %}
-                                                            </td>
-                                                        {% endfor %}
-                                                    </tr>
-                                                {% endfor %}
+                                            <table id="vuln_endpoints" class="table-striped table ">
```

Comment:
is this a typo?

Reply:
don't know what happened, but I selected a different line/section. There's an occurrence of 'ndpoints' in there somewhere.

Reply:
I thought you meant the whole block for second! Yes I found it. Good eyes. Holding off on committing in case you find something else.